src/tools/ScenarioMeasurement/Util/ProcessHelper.cs

```diff
@@ -108,6 +108,10 @@ public Result Run()
                     sw.WriteLine("Standard error:");
                     sw.WriteLine(error.ToString());
                 }
+                if (process.ExitCode != 0)
```

Comment:
If we're going to assert exit codes, we need a configuration to know what valid exit codes are. For example, robocopy returns a 1 if everything worked and a 0 if there were no errors, but no copying was done. [Source](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy#exit-return-codes)

Reply:
That makes sense! Is there any other way to detect the failure instead of giving a false positive result?

Reply:
Not really. Historically when we've valued exit codes we've defaulted to 0 being passing and optionally taking in a list of other acceptable exit codes. We could do something like that here if we wanted to. Did you encounter a situation where we returned non-0 and would have returned Success?

Reply:
Not yet. Currently we are running only dotnet.exe for the main iteration and cleanup and most likely it only returns 0 for success. If Startup Tool is only used and maintained by us, it seems checking 0 is enough.